go.mod

```diff
@@ -2,8 +2,6 @@ module go.infratographer.com/load-balancer-api
 
 go 1.20
 
-replace github.com/testcontainers/testcontainers-go => github.com/testcontainers/testcontainers-go v0.0.0-20230503220718-0df60de8ccd8
```

Comment:
Don't think we're ready to remove this quite yet per https://github.com/infratographer/load-balancer-api/pull/127#pullrequestreview-1415530704

Reply:
reverted back to as it was